UnitTests/GitCommandsTests/Git/GitTagControllerTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿using GitCommands.Git;
+using GitUIPluginInterfaces;
+using NSubstitute;
+using NUnit.Framework;
+using System;
+using TestMethod = NUnit.Framework.TestAttribute;
+
+namespace GitExtensionsTest.GitCommands.Git
+{
+    class GitTagControllerTest
+    {
+        private const string TagName = "bla";
+        private const string Revision = "0123456789";
+        private const string TagMessage = "foo";
+        private const string KeyId = "A9876F";
+        private const string WorkingDir = "C:\\";
```

Comment:
I think to avoid AAE and account for developer's paths, this should point to NUnit's TestDirectory
http://nunit.org/docs/2.6.2/testContext.html

Reply:
Changed, I've some problem with Test environment on Visual studio, so don't know if it works.